koku/masu/util/aws/common.py

```diff
@@ -871,6 +870,9 @@ def clear_s3_files(
                     exc_info=err,
                 )
     delete_s3_objects(request_id, to_delete, context)
+    manifest_accessor = ReportManifestDBAccessor()
+    manifest = manifest_accessor.get_manifest_by_id(manifest_id)
+    manifest_accessor.mark_s3_parquet_cleared(manifest)
```

Comment:
Can we add a comment to highlight that this is the only reason we are not deleting all parquet files later on in the process.